pages.fr/common/csh.md/csh.md

```diff
@@ -0,0 +1,21 @@
+# csh
+
+> Le shell (interprèteur de commandes) affiche une syntaxe proche de C.
+> Voir également : tcsh.
```

Comment:
```suggestion
> Voir aussi : tcsh.
```

[This](https://github.com/search?q=repo%3Atldr-pages%2Ftldr+Voir+aussi&type=code) seems to be used [more than "Voir également"](https://github.com/search?q=repo%3Atldr-pages%2Ftldr%20Voir%20%C3%A9galement&type=code) across our French pages.